### ISSUE TITLE
dank null voiding items.

### ISSUE THREAD

**Issue by juanitierno** [Cazadorsniper/PO3#52]
I crafted a diamond dank null. I put some logs and cobble in it, and then used a barrel several times to get stacks of items. 2 Unexpected things happened:
1) The items went into my inv, not danknull.
2) i dropped from inv, and the danknull seemed to collect them, then I fell to the nether, and the danknull was totally empty (not even the initial blocks I placed myself)
3) When veinmining soul sand in the nether with a tinkers shovel, it seemed to void the veinmined items.

I have added refined storage and torchmaster, in case it matters.

**Comment by sLpFhaWK** [comment]
I noticed this issue last night as well. I created all the Dank/Null's and went to use the Emerald one as the primary because, why not? So I added all the types of blocks I wanted to keep, Cobble, Wood, Nether, Obsidian etc and I had stacks and stacks of them in there. a few minutes pass and I go to the twilight forest to get some dirt and all my excessive items that i had 1k for in the dank are all capped at 64.

I haven't even tried mining yet as I just wanted to add blocks to store to keep out of my ME system but this won't work as I just LOST a bunch of mats.
